### DISCUSSION TITLE
Compatibility with Precor treadmills

### DISCUSSION THREAD

**Discussion by homedicare** [cagnulein/qdomyos-zwift#2432] [General]
Precor new 800 line with the touchscreen console. It can pair with bluetooth heart rate straps but I am not sure if it has FTMS. Is there any way to make it work?

**Comment by cagnulein** [comment]
@homedicare send me a nrf connect screenshot about it and I will check it. Nrfconnect is a free app on the store

**Reply by homedicare** [reply]
Despite turning the machine on and off in a 80+ bluetooth environment (gym), I cant find anything on the nrf connect.

The machine console is a P94 console from Precor / Peloton interactive. It runs android and is locked to certain functions only after account creation from the Precor end. Before creation I could access the android settings screen. It can only receive HRM straps on bluetooth. It does have a USB-C port on the motherboard.

**Comment by cagnulein** [comment]
Did you try limiting the reception only to very high rssi (near devices)?

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno lun 8 lug 2024 alle 18:37 homedicare ***@***.***>
ha scritto:

**Comment by homedicare** [comment]
yes that was done, no obvious signal.

**Comment by cagnulein** [comment]
did you check the datasheet of the precor?

**Comment by homedicare** [comment]
yes, "CSAFE ready, Bluetooth Pairing: 
Pair headphones and select heart rate straps or devices. Tap the Add Bluetooth
button and allow it to discover your device."

**Comment by cagnulein** [comment]
OK, so for sure it can't advertise itself. You can always control and read
the metrics from this treadmill using my electronic hack, but since this is
a treadmill on the gym, I guess you can do it, right? ;)

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno lun 8 lug 2024 alle 20:47 homedicare ***@***.***>
ha scritto:

**Comment by homedicare** [comment]
Yes I think so, where to start?

**Reply by cagnulein** [reply]
you have to use this https://www.amazon.it/dp/B08JV9J36T?ref_=cm_sw_r_apan_dp_CW1G5HRKM3G2HE21MJJF&language=it-IT soldiering each rely with the speed + and - and inclination + and - of the treadmill
